### DISCUSSION TITLE
Support saving of RGBA images with `mi.util.write_bitmap`

### DISCUSSION THREAD

**Discussion by maxfrei750** [mitsuba-renderer/mitsuba3#767] [Feature requests] [answered]
If I didn't make a mistake, then apparently `mi.util.write_bitmap` saves RGBA image as RGB, correct? If so, then I would propose to support the saving of RGBA images.

**Comment by njroussel** [comment, marked as answer]
Hi @maxfrei750 

The `mi.util.write_bitmap` function only expects RGB input. It's meant as a convenient and succinct one-liner for the most common case.
In fact, it usually only amounts to:
```python
mi.Bitmap(data).convert(
    pixel_format=mi.Bitmap.PixelFormat.RGB,
    component_format=mi.Struct.Type.UInt8,
    srgb_gamma=True
).write('filename.jpg', quality=-1)
```

The full interface is obviously more verbose and flexible. I still think this is a reasonable interface for end-users when `mi.util.write_bitmap` doesn't cover their needs. I don't see the need to further extend this helper function. For an additional alpha channel, only the `PixelFormat` needs to be changed.

**Reply by maxfrei750** [reply]
Thanks for the swift reply and especially for the additional information! You're absolutely right. This interface (which I wasn't aware of) covers everything a user (or at least I) might need. Very nice!
